### PR TITLE
TW-415 Can not click on invite in Chat settings

### PR DESCRIPTION
**Pull request by drminh2807**
# Issue
- #415 

# Demo

https://github.com/linagora/twake-on-matrix/assets/12546908/15280349-99be-446e-b1ed-4e8543e0d34d

https://github.com/linagora/twake-on-matrix/assets/12546908/5611638e-1436-4a77-86b5-f0942e51b12c

### CODE REVIEW

File: lib/utils/extension/build_context_extension.dart [hunk 1]
```diff
@@ -137,4 +138,9 @@ extension ContextExtensionss on BuildContext {
       return mobile!;
     }
   }
+
+  void goChild(String path) => go('${GoRouterState.of(this).uri.path}$path');
+
+  void pushChild(String path) =>
+      push('${GoRouterState.of(this).uri.path}$path');
```
Comment (nqhhdev) [inline lib/utils/extension/build_context_extension.dart:145]: What happens when `path` is null?

Comment (sherlockvn) [inline lib/utils/extension/build_context_extension.dart:145]: Why dont you `push('${GoRouterState.of(this).uri.path}/$path');`

Comment (drminh2807) [reply lib/utils/extension/build_context_extension.dart:145]: it not null

Comment (drminh2807) [reply lib/utils/extension/build_context_extension.dart:145]: no need,

**Review by hoangdat — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

File: lib/utils/extension/build_context_extension.dart (hunk 1 above)
Comment (sherlockvn) [reply lib/utils/extension/build_context_extension.dart:145]: i mean, so that you can use `pushChild('invite')` instead of `pushChild('/invite')`

Comment (hoangdat) [reply lib/utils/extension/build_context_extension.dart:145]: good idea

**Review by hoangdat — APPROVED** [review]

**Pull request merged**